render/hyperscript.js

```diff
@@ -28,6 +28,18 @@ function execSelector(state, attrs, children) {
 	var hasAttrs = false, childList, text
 	var className = attrs.className || attrs.class
 
+	if (Object.keys(state.attrs).length && Object.keys(attrs).length) {
```

Comment:
Note to self for later (if nobody beats me to it): introduce a more optimized helper here instead to check if an object has any properties.

```js
function isEmpty(object) {
    for (var key of object) if (hasOwn.call(object, key)) return true
    return false
}
```

(Nothing mandatory on your part @s-ilya)